Sources/VaporJWT/ConfigError.swift

```diff
@@ -0,0 +1,26 @@
+/// Errors that can take place during the config
+/// of required JWT entities such as the signer
+public struct ConfigError: Swift.Error {
+    var code: Int
```

Comment:
I think these properties should probably be public, and also `let` or `public internal(set)`